.eslintrc.js

```diff
@@ -1,18 +1,21 @@
 module.exports = {
   extends: [
-    'plugin:@shopify/typescript',
-    'plugin:@shopify/typescript-type-checking',
-    'plugin:@shopify/react',
-    'plugin:@shopify/jest',
-    'plugin:@shopify/prettier',
+    'plugin:@sewing-kit/typescript',
```

Comment:
Where do these configs live?

Reply:
[Here](https://github.com/lemonmade/sewing-kit/tree/main/packages/eslint-plugin/src/config), I believe.

Reply:
Found it, https://github.com/lemonmade/sewing-kit/tree/main/packages/eslint-plugin/src/config

Not related to quilt. It's interesting that there's a`@sewing-kit/eslint-plugin` vs `@sewing-kit/plugin-eslint`. One is a plugin for sewing-kit and one is an eslint plugin.

Reply:
Sorry, github is being wonky with this big PR. Comment thread didn't' refresh and the page is freezing.